Makefile.in

```diff
@@ -26,7 +26,7 @@ FB_LIBS =
 SVGA_OBJS = sys/svga/svgalib.o sys/pc/keymap.o @JOY@ @SOUND@
```

Comment:
please no gratuitous modechanges, thanks.
(refering to  Makefile.in 100644 → 100755, and the same for almost all other files)